common/notification/notification.ts

```diff
@@ -303,4 +314,57 @@ function diff(prev: any, curr: any, depth = 0) {
     return result;
 }
 
+export async function importMessageTranslations(messageTranslations: MessageTranslationFromDb[], overwrite = false) {
```

Comment:
For me, `overwrite` is a little bit misleading, because it sounds like we are only opting in to overwrite existing entries with new data. But in this case we are erasing all the entries, which is more like a force sync. I like Prisma's naming

```
    synchronize(dropBeforeSync?: boolean): Promise<void>;
```